tests/pyccel/scripts/exits/positive_exit3.py

```diff
@@ -0,0 +1,7 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+
+import sys
+
+if __name__ == "__main__":
+    exit_code = 2147483647
+    sys.exit(exit_code)
```

Comment:
Could you modify one of these tests to use `sys.exit`'s keyword please

Reply:
By this I mean a test that does:
```
sys.exit(status=exit_code)
```
The tests you have added are also useful but to make codacy happy you will need to do
```
from sys import exit as sys_exit
```

Reply:
so about that the `sys.exit` doesn't take any keyword arguments so for `python` it raises `TypeError`

Reply:
Huh looks like you're right: https://docs.python.org/3/library/sys.html#sys.exit

I guess they forgot to update their docstring
![image](https://user-images.githubusercontent.com/25363473/193242524-ed386eca-2c3a-4636-9d5f-6595b62d42dd.png)

In that case you can ignore this comment 👍

Reply:
okay done